lang/wp_smaily-et.po

```diff
@@ -78,67 +78,71 @@ msgid "Subdomain"
 msgstr "Alamdomeen"
 
 #: html/admin/html/form.php:31
-msgid "Username"
-msgstr "Kasutajatunnus"
+msgid "API Username"
+msgstr "API Kasutajatunnus"
 
 #: html/admin/html/form.php:35
-msgid "Password"
-msgstr "Salasõna"
+msgid "API Password"
+msgstr "API Salasõna"
 
 #: html/admin/html/form.php:40
+msgid "How to create API credentials?"
+msgstr "Kuidas luua API volikirjad?"
```

Comment:
"volikirjad" phrase does not make sense in the context of the plugin. It would be better to use just "Kuidas luua API ligipääs?" or "Kuidas luua API kasutaja?".

Reply:
I think kasutaja is much more easier to understand than volikirjad, added